### DISCUSSION TITLE
Refraction Shader error

### DISCUSSION THREAD

**Discussion by Psi25** [bnpr/Malt#593] [Q&A] [answered]
this code from https://github.com/bnpr/Malt/discussions/179

vec4 sample_opaque(vec2 uv)
{
    #ifdef PIXEL_SHADER
    {
        if(Settings.Transparency)
        {
            return texture(IN_OPAQUE_COLOR, uv);
        }
    }
    #endif
    return vec4(0);
}

is giving me error

![image](https://github.com/user-attachments/assets/e6212eb5-3c67-4ba5-abca-63bf94b8064f)

any help would be appreciated

**Comment by pragma37** [comment, marked as answer]
See #238
